tests/discussions.test.js

```diff
@@ -0,0 +1,364 @@
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Discussions Tracing Bad Request", (done) =>{
+        chai.request(server).get("/discussions/tracing/6168cf9")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+});
+
+describe("GET Tests Discussion",() =>{
+    before((done) =>{
+        const login = {
+            username: "MiroStar",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Discussions/ID", (done) =>{
+        chai.request(server).get("/discussions/616b0efeba862c9a697da9db")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("object");
+            response.body.should.have.property("_id");
+            response.body.should.have.property("comment");
+            response.body.should.have.property("title");
+            response.body.should.have.property("theme");
+            response.body.should.have.property("idAccount");
+            response.body.should.have.property("dateCreation");
+            response.body.should.have.property("status");
+            done();
+        });
+    });
+
+    it("GET /Discussions/ID Not Found", (done) =>{
+        chai.request(server).get("/discussions/616b0efeba862c9a697da9dc")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Discussions/ID Bad Request", (done) =>{
+        chai.request(server).get("/discussions/616b0efeba862c9a697")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+});
+
+describe("GET Tests Discussions",() =>{
+    before((done) =>{
+        const login = {
+            username: "MiroStar",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Discussions", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("GET /Discussions", (done) => {
```